content/docs/add-react-to-an-existing-app.md

```diff
@@ -6,91 +6,92 @@ prev: add-react-to-a-new-app.html
 
->**Note:**
->
->Once installed, we strongly recommend setting up a [production build process](/docs/optimizing-performance.html#use-the-production-build) to ensure you're using the fast version of React in production.
-
-We recommend using [Yarn](https://yarnpkg.com/) or [npm](https://www.npmjs.com/) for managing front-end dependencies. If you're new to package managers, the [Yarn documentation](https://yarnpkg.com/en/docs/getting-started) is a good place to get started.
-
-To install React with Yarn, run:
-
-```bash
-yarn init
-yarn add react react-dom
+```html
+<script
+  crossorigin
+  src="https://unpkg.com/react@16/umd/react.development.js">
+</script>
+<script
+  crossorigin
+  src="https://unpkg.com/react-dom@16/umd/react-dom.development.js">
+</script>
 ```
 
-To install React with npm, run:
+### 2. Write your components
 
-```bash
-npm init
-npm install --save react react-dom
+```js
+// src/button.js
+
+class Button extends React.Component {
+  render() {
+    return (
+      <button className="btn">{this.props.label}</button>
+    );
+  }
+}
 ```
 
-Both Yarn and npm download packages from the [npm registry](http://npmjs.com/).
-
->  Note:
->
->  To prevent potential incompatibilities, all react packages should use the same version. (This includes `react`, `react-dom`, `react-test-renderer`, etc.)
-
-### Enabling ES6 and JSX
-
-We recommend using React with [Babel](http://babeljs.io/) to let you use ES6 and JSX in your JavaScript code. ES6 is a set of modern JavaScript features that make development easier, and JSX is an extension to the JavaScript language that works nicely with React.
-
-The [Babel setup instructions](https://babeljs.io/docs/setup/) explain how to configure Babel in many different build environments. Make sure you install [`babel-preset-react`](http://babeljs.io/docs/plugins/preset-react/#basic-setup-with-the-cli-) and [`babel-preset-env`](http://babeljs.io/docs/plugins/preset-env/) and enable them in your [`.babelrc` configuration](http://babeljs.io/docs/usage/babelrc/), and you're good to go.
-
-### Hello World with ES6 and JSX
-
-We recommend using a bundler like [webpack](https://webpack.js.org/) or [Browserify](http://browserify.org/), so you can write modular code and bundle it together into small packages to optimize load time.
+```js
+// Render in several places
 
-The smallest React example looks like this:
+ReactDOM.render(
+  <Button label="Sign Out" />,
+  document.querySelector('.sign-out')
+);
 
-```js
-import React from 'react';
-import ReactDOM from 'react-dom';
+ReactDOM.render(
+  <Button label="Sign In" />,
+  document.querySelector('.sign-in')
+);
 
 ReactDOM.render(
-  <h1>Hello, world!</h1>,
-  document.getElementById('root')
+  <Button label="Edit" />,
+  document.querySelector('.edit')
 );
 ```
 
-This code renders into a DOM element with the id of `root`, so you need `<div id="root"></div>` somewhere in your HTML file.
+### 3. Compile with Babel
```

Comment:
we don't need babel to compile, we can require inline, the whole point of this piece is that we can use babel in a script:

```
// babel in a script

<script src="https://unpkg.com/babel-standalone@6.26.0/babel.js"></script>

// lower down

<script type="text/babel">
```

Reply:
That's extremely slow - this is meant for people adding React to an actual app. If they are just trying React they should be referring to the Try React page.